sdk/identity/identity/test/internal/node/clientCertificateCredential.spec.ts

```diff
@@ -114,7 +114,7 @@ describe("ClientCertificateCredential (internal)", function () {
   });
 
   it("throws when given a file that doesn't contain a PEM-formatted certificate", async function (this: Context) {
-    const fullPath = path.resolve(__dirname, "../src/index.ts");
+    const fullPath = path.resolve("./clientCertificateCredential.spec.ts");
```

Comment:
__dirname is not supported, but it doesn't actually matter what file we send as long as it's not a proper cert

Reply:
yeah __dirname is blocking some packages' test too.  There's recommendation of  `const __dirname=path.dirname(url.fileURLToPath(import.meta.url)` but `import.meta.url` requires "module" of "ES2020", which is at odds with our module of commonjs for test:node-ts-input :(